internal/metrics/metricsManager.go

```diff
@@ -39,6 +40,10 @@ const (
 	counterFilesystemUsagePercentage              = "assisted_installer_filesystem_usage_percentage"
 	histogramMonitoredHostsDurationMs             = "assisted_installer_monitored_hosts_duration_ms"
 	histogramMonitoredClustersDurationMs          = "assisted_installer_monitored_clusters_duration_ms"
+	counterInstallerCacheGetRelease               = "assisted_installer_cache_get_release"
+	counterInstallerCacheReleaseCached            = "assisted_installer_cache_get_release_cached"
```

Comment:
I think this is unused

Reply:
I don't think so...

Reply:
We need the one for cached as far as I can tell...

Reply:
I removed `counterInstallerCacheGetRelease` 

Though `counterInstallerCacheReleaseCached` is still used.